### ISSUE TITLE
oAuth2 stuck

### ISSUE THREAD

**Issue by steklopod** [nuxt-alt/auth#85]
### Environment

- Node Version:     v20.8.1
- Nuxt Version:     3.8.2
- CLI Version:      3.10.0
- Nitro Version:    2.8.0
- Package Manager:  pnpm@8.11.0
- Builder:          -
- User Config:      experimental, ssr, runtimeConfig, build, css, hooks, modules, auth, app
- Runtime Modules:  @nuxt-alt/auth@3.0.3, @pinia/nuxt@0.5.1, nuxt-vitest@0.11.5, @nuxtjs/stylelint-module@5.1.0, @vueuse/nuxt@10.5.0
- Build Modules:    -

### Nuxt Config

```js
  auth: {
    scopeKey: 'scope',
    globalMiddleware: true,
    // routerStrategy: 'navigateTo',
    strategies: {
      refresh: {
        scheme: 'refresh'
   }
}
```

### Reproduction

https://colaba.online/login?t=0

<img width="392" alt="Снимок экрана 2023-12-04 в 13 56 54" src="https://github.com/nuxt-alt/auth/assets/29500153/5d522c82-267b-4240-9b62-c162f115ed02">

Just press the button and try to login

___

### Describe the bug

I have backend that manages oAuth2. 

After success it sets cookies and redirect to main frontend page

<img width="1202" alt="Снимок экрана 2023-12-04 в 13 56 17" src="https://github.com/nuxt-alt/auth/assets/29500153/9393e3d6-9359-4e72-abbc-84e4729cd101">

In latest versions of library if Cookies a set it breaks entire application: redirect will be never finished and node stops working.

### Additional context

- `username+password` works.
- no problems on backend

### Desired behaviour

If cookies were set by backend and I was redirected to main page:
- log in
- fetch user

>  This worked earlier. Maybe latest changes with `watchLoggedIn` causes this 🤷🏻

**Comment by Denoder** [comment]
Accessing the url itself is loading extremely slow without me interacting, so im uncertain if this pertains to the module at all.

**Comment by steklopod** [comment]
I always use `refresh` scheme. It worked before in older versions. 

I mentioned `cookie` scheme to show how I tried to solve the problem by adding second strategy. It did not help.

I don't need Google authentication flow implemented on frontend. I do it on backend. I just reported that earlier the functionality allowed automatic `login if user was not loggedIn but token is not empty in cookies`.


So let me try to explain it with pseudo-code of auto-login:
```js
onBeforeMount(() => {
  if (!useAuth().loggedIn && cookies.token.isNotEmpty() && cookies.token.isNotExpired()){
    await useAuth().login() // + fetch user
  }
})
```
